operations/app/src/environments/tf

```diff
@@ -165,6 +183,18 @@ if [[ "${ACTUAL_RESULTS?}" == "${EXPECTED_RESULTS?}" ]]; then
         ACTUAL_RESULTS="${ACTUAL_RESULTS?}${?}"
         ;;
 
+    plan-file*)
+        # Custom command we made for the purpose of CI/CD
+        EXPECTED_RESULTS="${EXPECTED_RESULTS?}0"
+        terraform plan \
+            --var-file "${TFVARS?}" \
+            >/dev/null -out=${PLAN_DIR?}/${ENVIRONMENT?}-${HERE?}.tfplan
```

Comment:
I did not know that you could redirect stdout in the middle of the command (so TIL :) ); I would however prefer it at the end (and not chop up the command in the middle)

Reply:
Consider explicitly redirecting both stdout and stderr:
`1>/dev/null 2>/dev/null`
or maybe dump it in a file that is uploaded into Azure storage?

Reply:
Will add a `1>.stdout` and `2>.stderr` as an output.

Reply:
Fixed in next push